File: Python/Fundamentals/04_Functions/Exercise/odd_and_even-sum.py
input_number = input()


def odd_even_sum(number) -> object:
    even_sum = 0
    odd_sum = 0
    for i in range(0, len(number)):
        if int(number[i]) % 2 == 0:
            even_sum += int(number[i])
        else:
            odd_sum += int(number[i])

    print(f'Odd sum = {odd_sum}, Even sum = {even_sum}')


odd_even_sum(input_number)

File: JS/JS  Fundamentals/04_Functions/exercise/oddAndEvenSum.js
function solve(number) {
    const numberToString = String(number);
    let oddSum = 0;
    let evenSum = 0;
    
    for (let n of numberToString) {
        const  num = Number(n);
        if (num % 2 == 0) {
            evenSum += num;
        } else {
            oddSum += num
        }
    }

    return `Odd sum = ${oddSum}, Even sum = ${evenSum}`
}


console.log(solve(1000435));
console.log(solve(3495892137259234));